engines/sword25/detection_tables.h

```diff
@@ -105,6 +105,15 @@ static const ADGameDescription gameDescriptions[] = {
 		ADGF_NO_FLAGS,
 		GUIO1(GUIO_NOASPECT)
 	},
+	{
+			"sword25",
+			"",
+			AD_ENTRY1s("lang_he.b25c", "7b0e58798793e0dd87023ef112a11d0f", 461034),
```

Comment:
Where can I get this file?

Reply:
The file will be release after this PR merge in public facebook group (of adventure game fans) [in this group ](https://he-il.facebook.com/groups/200491360554968).
For now the file saved in my cloud storage.
Do you need access to this file?

Reply:
I'd like to have it later, so we put on our website.